trunk/SUAVE/Attributes/Propellants/JP_7.py

```diff
@@ -0,0 +1,57 @@
+## @ingroup Attributes-Propellants
+#Jet A
+#
+# Created:  Unk 2013, SUAVE TEAM
+# Modified: Feb 2016, M. Vegh
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+from Propellant import Propellant
+
+# ----------------------------------------------------------------------
+#  Jet_A Propellant Class
+# ----------------------------------------------------------------------
+## @ingroup Attributes-Propellants
+class JP_7(Propellant):
+    """Holds values for this propellant
+    
+    Assumptions:
+    None
+    
+    Source:
+    None
+    """
+
+    def __defaults__(self):
+        """This sets the default values.
+
+        Assumptions:
+        None
+
+        Source:
+        Values commonly available
+
+        Inputs:
+        None
+
+        Outputs:
+        None
+
+        Properties Used:
+        None
+        """    
+        self.tag                       = 'JP_7'
+        self.reactant                  = 'O2'
+        self.density                   = 820.0                          # kg/m^3 (15 C, 1 atm)
```

Comment:
Maybe you can specify the units throughout all of your defaults like this:
* Units['kg/m**3']

I think this is more descriptive and more inline with SUAVE style.